.test-infra/jenkins/job_PostCommit_Python_MongoDBIO_IT.groovy

```diff
@@ -32,6 +32,7 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Python_MongoDBIO_IT',
     gradle {
       rootBuildScriptDir(commonJobProperties.checkoutDir)
       tasks(':sdks:python:test-suites:direct:py2:mongodbioIT')
+      tasks(':sdks:python:test-suites:direct:py37:mongodbioIT')
```

Comment:
If we test only one minor version, I suggest python 3.5 as this is the lowest version beam supports.

Reply:
will change to that.